Twig/Extension/AceEditorExtension.php

```diff
@@ -71,7 +71,7 @@ public function getFunctions()
 
     public function includeAceEditor()
     {
-        if (!$this->environment->hasExtension('assets')) {
+        if (!$this->environment->hasExtension('asset')) {
```

Comment:
!$this->environment->hasExtension('asset') && !$this->environment->hasExtension('assets') 

make it compatible with older versions